gradle-suppressible-error-prone/src/test/groovy/com/palantir/gradle/suppressibleerrorprone/SuppressibleErrorPronePluginIntegrationTest.groovy

```diff
@@ -41,6 +41,18 @@ class SuppressibleErrorPronePluginIntegrationTest extends ConfigurationCacheSpec
 
         // language=Gradle
         buildFile << '''
+            buildscript {
+                repositories {
+                    mavenCentral()
+                }
+                dependencies {
+                    classpath 'com.palantir.gradle.consistentversions:gradle-consistent-versions:3.1.0'
```

Comment:
yeah, I am struggling to see a reason to not have gcv on stuff generally, I too have written a number of tests that just would not work where it's deployed.

Reply:
Yeah, every plugin should have GCV, it interacts with everything. Perhaps we should make sure any upcoming testing framework we have includes it by default.